src/window/main_menu.c

```diff
@@ -34,12 +38,33 @@ static generic_button buttons[] = {
 static image_button movie_button =
     {591, 442, 33, 22, IB_NORMAL, 89, 0, button_intro_movie, button_none, 0, 0, 1};
 
+static void draw_version_string(void)
+{
+    char version_string[100] = "v";
+    int version_length = string_length(JULIUS_VERSION);
+    int text_y = screen_height() - 30;
+
+    string_copy(JULIUS_VERSION, version_string + 1, 99);
```

Comment:
Rebase and use the new `string_from_ascii()` to get rid of the compile warnings.